src/test/ensemble_tests.cpp

```diff
@@ -88,7 +88,7 @@ class EnsembleFlowBothApiTest : public TestWithTempDir {
         prepareKFSInferInputTensor(request, customPipelineInputName, std::make_tuple(shape, ovmsPrecisionToKFSPrecision(ovms::Precision::FP32)), requestData);
     }
 
-    void checkDummyResponse(int seriesLength, int batchSize = 1, std::string servableName = "") {
+    void checkDummyResponse(int seriesLength, int batchSize = 1, std::string& servableName = "") {
```

Comment:
`checkDummyResponse` should not change `servableName` parameter inside its implementation, therefore I suggest changing to const, even though its only unit test